### PR TITLE
Update Train.md - Removing Expand/Collapse button

### PR DESCRIPTION
**Pull request by tttttanya**
Removing Expand/Collapse button to keep the page in sync

# Description 

- [x] Documentation update 

## Checklist

- [x] I have signed the [Contributor License Agreement](https://github.com/tensorflow/models/wiki/Contributor-License-Agreements).
- [x] I have read [guidelines for pull request](https://github.com/tensorflow/models/wiki/Submitting-a-pull-request).
- [x] My code follows the [coding guidelines](https://github.com/tensorflow/models/wiki/Coding-guidelines).
- [x] I have performed a self [code review](https://github.com/tensorflow/models/wiki/Code-review) of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

File: official/nlp/docs/train.md
Comment (Chen-Zhe) [inline official/nlp/docs/train.md:162]: You can remove the 2 empty lines after removing the tag.

**Review by Chen-Zhe — CHANGES_REQUESTED** [review]
LGTM after suggested change

File: official/nlp/docs/train.md
Comment (tttttanya) [reply official/nlp/docs/train.md:162]: Removed extra lines

**Comment by saberkun** [comment]
Let's add ready-to-pull after the approval.

**Review by Chen-Zhe — APPROVED** [review]

**Pull request merged**
